### DISCUSSION TITLE
Atari VCS Stick - BlueRetro Core

### DISCUSSION THREAD

**Discussion by lamprey13** [darthcloud/BlueRetro#778] [General]
Hi,

I have a couple of the Atari VCS Sticks I'd like to use on some consoles, primarily on the 2600.

Does anyone know if the Atari VCS Sticks will pair with the BlutReto Core?
https://www.blue-retro.com/products/core/

If so, then I could get the Sega Genesis adapter and hook it to my 2600. But, I'm having a hrd time finding information about pairing the VCS sticks.

Thanks!

**Comment by DoomyDoomer** [comment]
I can't speak to the sticks working or not but the VCS controller doesn't. Best bet is to send Darth an HCI Trace for the VCS Stick. Here's instructions on how to do that: https://github.com/darthcloud/BlueRetro/wiki/Bluetooth-HCI-trace-with-Win10. I've been meaning to do the same for the VCS Controller.

**Comment by lamprey13** [comment]
Ahh, nice. I'll see if I can get a trace this weekend!
